docs/gpu-support.md

```diff
@@ -0,0 +1,73 @@
+# Deploying with Accelerators in the Cloud HPC Toolkit
+
+## Supported modules
+
+* [vm-instance]
+* Slurm on GCP modules where applicable, both version 4 and version 5
+  * `schedmd-slurm-gcp-v5-*`
+  * `SchedMD-slurm-on-gcp-*`
+* PBS Pro modules (`pbspro-*`)
+
+## Accelerator definition automation
+
+The [vm-instance] and schedmd-slurm-gcp-v5 [node-group], [controller] and
+[login] modules support automation for defining the `guest_accelerator` config.
+If the user supplies any value for this setting, the automation will be bypassed.
```

Comment:
What about modules that use vm-instance? should we not mention them here?

Reply:
Done

Reply:
End users will probably not know what modules use it. I like it how you. mentioned them by name up in the first section and I would repeat that here, even if just like:
```
(...) relying on [vm-instance] (htcondor-*, omnia-install, pbs-pro-*) (...)
```

Reply:
Done